src/modules/nfts/components/wizard/RoyaltiesCreators.tsx

```diff
@@ -360,9 +360,7 @@ export default function RoyaltiesCreators({
   const previousNFT: NFTFormValue | undefined = nftList[`nft-${index - 1}`];
 
   const [creators, setCreators] = useState<Array<Creator>>(
-    previousNFT
-      ? previousNFT.properties.creators
-      : [HOLAPLEX_CREATOR_OBJECT, { address: userKey ?? '', share: 98 }]
+    previousNFT ? previousNFT.properties.creators : [{ address: userKey ?? '', share: 98 }] // removing the first array item 'HOLAPLEX_CREATOR_OBJECT' for now as a hot fix to a bug preventing minting in production. @TODO -- figure out what is going on
```

Comment:
Really. this has been in place for quite some time. at least 2 weeks. 

We need to keep the user as the first creator.